code/dynamic_programming/Array_Median/med.c

```diff
@@ -0,0 +1,44 @@
+#include<stdio.h>
+
+#define N 10                                                
+                                                               
+  int main( )                                                     
+   {                                                           
+       int i,j,n;                                              
+       float median,a[N],t;                                    
+                                                               
+       printf("Enter the number of items\n");                  
+       scanf("%d", &n);                                        
+    /* Reading items into array a  */
```

Comment:
This code will not work if the use enters a value for `n` that is greater than 10, since that is what N is set to. You should create a dynamic array using `malloc`, in which the size is determined at runtime, so this works well.